2024-10-19T22:48:54-05:00 INF Traefik version 3.1.6 built on 2024-10-09T13:57:41Z version=3.1.6
2024-10-19T22:48:54-05:00 INF 
Stats collection is disabled.
Help us improve Traefik by turning this feature on :)
More details on: https://doc.traefik.io/traefik/contributing/data-collection/

2024-10-19T22:48:54-05:00 INF Starting provider aggregator aggregator.ProviderAggregator
2024-10-19T22:48:54-05:00 INF Starting provider *file.Provider
2024-10-19T22:48:54-05:00 ERR Cannot start the provider *file.Provider error="error adding file watcher: no such file or directory"
2024-10-19T22:48:54-05:00 INF Starting provider *traefik.Provider
2024-10-19T22:48:54-05:00 INF Starting provider *docker.Provider
2024-10-19T22:48:54-05:00 INF Starting provider *acme.ChallengeTLSALPN
2024-10-19T22:48:54-05:00 WRN Could not find network named "proxy" for container "/e-17-odb-1". Maybe you're missing the project's prefix in the label? container=odb-e-17-023f213fc046190bf8c873e7dac0452ba4848a94c52975c16ddb8a30a799e0d7 providerName=docker serviceName=odb-e-17
2024-10-19T22:48:54-05:00 WRN Defaulting to first available network (&{"odoo-net" "172.23.0.2" '\x00' "" "436965aa2584246f3a5eafbf3962cb3006a780b2e3604d7936d8dbc00fc49d17"}) for container "/e-17-odb-1". container=odb-e-17-023f213fc046190bf8c873e7dac0452ba4848a94c52975c16ddb8a30a799e0d7 providerName=docker serviceName=odb-e-17
2024-10-19T22:48:54-05:00 WRN Could not find network named "proxy" for container "/pihole". Maybe you're missing the project's prefix in the label? container=pihole-pihole-52c9f0ae5853038c69da8827618a74eb116e587a4ca9768df23e544d46671575 providerName=docker serviceName=pihole
2024-10-19T22:48:54-05:00 WRN Defaulting to first available network (&{"pihole_default" "172.22.0.2" '\x00' "" "ab6958033497927c8c09a01c6981f1064b66383a836a0c8dc847db02214d1bdd"}) for container "/pihole". container=pihole-pihole-52c9f0ae5853038c69da8827618a74eb116e587a4ca9768df23e544d46671575 providerName=docker serviceName=pihole
2024-10-19T22:48:54-05:00 WRN Could not find network named "proxy" for container "/e-17-odb-1". Maybe you're missing the project's prefix in the label? container=odb-e-17-023f213fc046190bf8c873e7dac0452ba4848a94c52975c16ddb8a30a799e0d7 providerName=docker serviceName=odb-e-17
2024-10-19T22:48:54-05:00 WRN Defaulting to first available network (&{"odoo-net" "172.23.0.2" '\x00' "" "436965aa2584246f3a5eafbf3962cb3006a780b2e3604d7936d8dbc00fc49d17"}) for container "/e-17-odb-1". container=odb-e-17-023f213fc046190bf8c873e7dac0452ba4848a94c52975c16ddb8a30a799e0d7 providerName=docker serviceName=odb-e-17
2024-10-19T22:48:54-05:00 WRN Could not find network named "proxy" for container "/pihole". Maybe you're missing the project's prefix in the label? container=pihole-pihole-52c9f0ae5853038c69da8827618a74eb116e587a4ca9768df23e544d46671575 providerName=docker serviceName=pihole
2024-10-19T22:48:54-05:00 WRN Defaulting to first available network (&{"pihole_default" "172.22.0.2" '\x00' "" "ab6958033497927c8c09a01c6981f1064b66383a836a0c8dc847db02214d1bdd"}) for container "/pihole". container=pihole-pihole-52c9f0ae5853038c69da8827618a74eb116e587a4ca9768df23e544d46671575 providerName=docker serviceName=pihole
2024-10-19T22:48:54-05:00 ERR error="middleware \"myauth@file\" does not exist" entryPointName=websecure routerName=traefik-secure@docker
2024-10-19T22:48:56-05:00 ERR error="middleware \"myauth@file\" does not exist" entryPointName=websecure routerName=traefik-secure@docker
2024-10-19T22:49:46-05:00 WRN Could not find network named "proxy" for container "/e-17-odb-1". Maybe you're missing the project's prefix in the label? container=odb-e-17-023f213fc046190bf8c873e7dac0452ba4848a94c52975c16ddb8a30a799e0d7 providerName=docker serviceName=odb-e-17
2024-10-19T22:49:46-05:00 WRN Defaulting to first available network (&{"odoo-net" "172.23.0.2" '\x00' "" "436965aa2584246f3a5eafbf3962cb3006a780b2e3604d7936d8dbc00fc49d17"}) for container "/e-17-odb-1". container=odb-e-17-023f213fc046190bf8c873e7dac0452ba4848a94c52975c16ddb8a30a799e0d7 providerName=docker serviceName=odb-e-17
2024-10-19T22:49:46-05:00 WRN Could not find network named "proxy" for container "/pihole". Maybe you're missing the project's prefix in the label? container=pihole-pihole-52c9f0ae5853038c69da8827618a74eb116e587a4ca9768df23e544d46671575 providerName=docker serviceName=pihole
2024-10-19T22:49:46-05:00 WRN Defaulting to first available network (&{"pihole_default" "172.22.0.2" '\x00' "" "ab6958033497927c8c09a01c6981f1064b66383a836a0c8dc847db02214d1bdd"}) for container "/pihole". container=pihole-pihole-52c9f0ae5853038c69da8827618a74eb116e587a4ca9768df23e544d46671575 providerName=docker serviceName=pihole
2024-10-19T22:49:46-05:00 INF I have to go...
2024-10-19T22:49:46-05:00 INF Stopping server gracefully
2024-10-19T22:49:46-05:00 ERR error="accept tcp [::]:443: use of closed network connection" entryPointName=websecure
2024-10-19T22:49:46-05:00 ERR error="accept tcp [::]:80: use of closed network connection" entryPointName=web
2024-10-19T22:49:46-05:00 ERR error="close tcp [::]:443: use of closed network connection" entryPointName=websecure
2024-10-19T22:49:46-05:00 ERR error="close tcp [::]:80: use of closed network connection" entryPointName=web
2024-10-19T22:49:46-05:00 INF Server stopped
2024-10-19T22:49:46-05:00 INF Shutting down
2024-10-19T22:49:46-05:00 ERR error="middleware \"myauth@file\" does not exist" entryPointName=websecure routerName=traefik-secure@docker
{"level":"info","version":"3.5.2","time":"2025-09-21T16:08:26-05:00","message":"Traefik version 3.5.2 built on 2025-09-09T10:17:00Z"}
{"level":"info","time":"2025-09-21T16:08:26-05:00","message":"\nStats collection is disabled.\nHelp us improve Traefik by turning this feature on :)\nMore details on: https://doc.traefik.io/traefik/contributing/data-collection/\n"}
{"level":"info","time":"2025-09-21T16:08:26-05:00","message":"Starting provider aggregator *aggregator.ProviderAggregator"}
{"level":"info","time":"2025-09-21T16:08:26-05:00","message":"Starting provider *file.Provider"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/middlewares.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:08:26-05:00","message":"Error while building configuration (for the first time)"}
{"level":"info","time":"2025-09-21T16:08:26-05:00","message":"Starting provider *traefik.Provider"}
{"level":"info","time":"2025-09-21T16:08:26-05:00","message":"Starting provider *acme.ChallengeTLSALPN"}
{"level":"info","time":"2025-09-21T16:08:26-05:00","message":"Starting provider *docker.Provider"}
{"level":"info","time":"2025-09-21T16:08:26-05:00","message":"Starting provider *acme.Provider"}
{"level":"info","providerName":"letsencrypt.acme","acmeCA":"https://acme-v02.api.letsencrypt.org/directory","time":"2025-09-21T16:08:26-05:00","message":"Testing certificate renew..."}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/middlewares.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:10:56-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/middlewares.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:10:56-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/middlewares.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:10:56-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/middlewares.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:18:55-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/middlewares.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:18:55-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/middlewares.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:18:55-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:20:06-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:20:06-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:20:06-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:20:06-05:00","message":"Error occurred during watcher callback"}
{"level":"info","time":"2025-09-21T16:20:30-05:00","message":"I have to go..."}
{"level":"info","time":"2025-09-21T16:20:30-05:00","message":"Stopping server gracefully"}
{"level":"info","time":"2025-09-21T16:20:30-05:00","message":"Server stopped"}
{"level":"info","time":"2025-09-21T16:20:30-05:00","message":"Shutting down"}
{"level":"info","version":"3.5.2","time":"2025-09-21T16:22:32-05:00","message":"Traefik version 3.5.2 built on 2025-09-09T10:17:00Z"}
{"level":"info","time":"2025-09-21T16:22:32-05:00","message":"\nStats collection is disabled.\nHelp us improve Traefik by turning this feature on :)\nMore details on: https://doc.traefik.io/traefik/contributing/data-collection/\n"}
{"level":"info","time":"2025-09-21T16:22:32-05:00","message":"Starting provider aggregator *aggregator.ProviderAggregator"}
{"level":"info","time":"2025-09-21T16:22:32-05:00","message":"Starting provider *file.Provider"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:22:32-05:00","message":"Error while building configuration (for the first time)"}
{"level":"info","time":"2025-09-21T16:22:32-05:00","message":"Starting provider *traefik.Provider"}
{"level":"info","time":"2025-09-21T16:22:32-05:00","message":"Starting provider *docker.Provider"}
{"level":"info","time":"2025-09-21T16:22:32-05:00","message":"Starting provider *acme.ChallengeTLSALPN"}
{"level":"info","time":"2025-09-21T16:22:32-05:00","message":"Starting provider *acme.Provider"}
{"level":"info","providerName":"letsencrypt.acme","acmeCA":"https://acme-v02.api.letsencrypt.org/directory","time":"2025-09-21T16:22:32-05:00","message":"Testing certificate renew..."}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:26:21-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:26:21-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:26:21-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:26:21-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:26:21-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:26:21-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:26:21-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:26:21-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:28:38-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:28:38-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:28:38-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T16:28:38-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T20:39:35-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T20:39:35-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T20:39:35-05:00","message":"Error occurred during watcher callback"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T20:39:35-05:00","message":"Error occurred during watcher callback"}
{"level":"info","time":"2025-09-21T21:43:40-05:00","message":"I have to go..."}
{"level":"info","time":"2025-09-21T21:43:40-05:00","message":"Stopping server gracefully"}
{"level":"info","time":"2025-09-21T21:43:41-05:00","message":"Server stopped"}
{"level":"info","time":"2025-09-21T21:43:41-05:00","message":"Shutting down"}
{"level":"info","version":"3.5.2","time":"2025-09-21T21:43:42-05:00","message":"Traefik version 3.5.2 built on 2025-09-09T10:17:00Z"}
{"level":"info","time":"2025-09-21T21:43:42-05:00","message":"\nStats collection is disabled.\nHelp us improve Traefik by turning this feature on :)\nMore details on: https://doc.traefik.io/traefik/contributing/data-collection/\n"}
{"level":"info","time":"2025-09-21T21:43:42-05:00","message":"Starting provider aggregator *aggregator.ProviderAggregator"}
{"level":"info","time":"2025-09-21T21:43:42-05:00","message":"Starting provider *file.Provider"}
{"level":"error","providerName":"file","error":"/etc/traefik/dynamic/tls.yaml: field not found, node: sslStrategies","time":"2025-09-21T21:43:42-05:00","message":"Error while building configuration (for the first time)"}
{"level":"info","time":"2025-09-21T21:43:42-05:00","message":"Starting provider *traefik.Provider"}
{"level":"info","time":"2025-09-21T21:43:42-05:00","message":"Starting provider *docker.Provider"}
{"level":"info","time":"2025-09-21T21:43:42-05:00","message":"Starting provider *acme.ChallengeTLSALPN"}
{"level":"info","time":"2025-09-21T21:43:42-05:00","message":"Starting provider *acme.Provider"}
{"level":"info","providerName":"letsencrypt.acme","acmeCA":"https://acme-v02.api.letsencrypt.org/directory","time":"2025-09-21T21:43:42-05:00","message":"Testing certificate renew..."}
